src/profile/components/multiItemAddressRow/MultiItemAddressRow.tsx

```diff
@@ -0,0 +1,189 @@
+                as={TextInput}
+                invalid={hasFieldError(formikProps, 'city')}
+                aria-invalid={hasFieldError(formikProps, 'city')}
+                helperText={getFieldErrorMessage(formikProps, 'city')}
+                labelText={t(formFields.city.translationKey)}
+                aria-labelledby={`${dataType}-city-helper`}
+              />
+              <div className={commonFormStyles.formField}>
+                <FormikDropdown
+                  name="countryCode"
+                  id={`${testId}-countryCode`}
+                  options={countryOptions}
+                  label={t(formFields.country.translationKey)}
+                  default={countryCode}
+                  onChange={option =>
+                    formikProps.setFieldValue(
+                      'countryCode',
+                      (option as HdsOptionType).value
+                    )
+                  }
+                />
+              </div>
+            </div>
+            <FormButtons
+              handler={actionHandler}
+              disabled={!!currentAction}
+              testId={testId}
+              alignLeft
+            />
+            <SaveIndicator action={currentAction} testId={testId} />
+          </Form>
+        )}
+      </Formik>
+    );
+  }
+  return (
+    <div
+      className={classNames([
+        commonFormStyles.contentWrapper,
+        commonFormStyles.multiItemContentWrapper,
+      ])}
+    >
+      <h4 className={commonFormStyles.sectionTitle}>
+        {primary
+          ? t('profileInformation.primaryAddress')
+          : t('profileInformation.address')}
+      </h4>
+      <div className={commonFormStyles.multiItemWrapper}>
+        <LabeledValue
+          label={t(formFields.address.translationKey)}
+          value={value.address}
+          testId={`${testId}-address`}
+        />
+        <LabeledValue
+          label={t(formFields.postalCode.translationKey)}
+          value={value.postalCode}
+          testId={`${testId}-postalCode`}
+        />
+        <LabeledValue
+          label={t(formFields.city.translationKey)}
+          value={value.city}
+          testId={`${testId}-city`}
+        />
+        <LabeledValue
+          label={t(formFields.country.translationKey)}
+          value={getCountry(value.countryCode, lang)}
+          testId={`${testId}-countryCode`}
+        />
+      </div>
+      <div className={commonFormStyles.actionsWrapper}>
+        <EditButtons
+          handler={actionHandler}
+          actions={{
+            removable: false,
+            primary,
+            setPrimary: false,
+          }}
+          buttonClassNames={commonFormStyles.actionsWrapperButton}
+          editButtonId={`${testId}-edit-button`}
+          disabled={!!currentAction}
+          testId={testId}
+        />
+        <SaveIndicator action={currentAction} testId={testId} />
+      </div>
+    </div>
+  );
+}
+
+export default MultiItemAddressRow;
```

Comment:
Should we change this export to inline?

Reply:
Following other exports in .tsx-files here.